migration/sql-files/084-type-groups-and-child-types.sql

```diff
@@ -39,4 +39,8 @@ CREATE TABLE work_item_child_types (
     position integer DEFAULT 0 NOT NULL
 );
 
-CREATE UNIQUE INDEX work_item_child_types_uidx ON work_item_child_types (parent_work_item_type_id, child_work_item_type_id) WHERE deleted_at IS NULL;
\ No newline at end of file
+CREATE UNIQUE INDEX work_item_child_types_uidx ON work_item_child_types (parent_work_item_type_id, child_work_item_type_id) WHERE deleted_at IS NULL;
+
```

Comment:
also, do we need to include space_template_id in the unique index?

Reply:
Good point again.

Reply:
@pranavgore09 currently the `work_item_child_types` table has no space template relation, so I cannot make the `space_template_id` a part of this key.